assignment1_permutation.py

```diff
@@ -0,0 +1,34 @@
+# Uses python3
```

Comment:
You can write:

# /usr/bin/env python3

to force use of the python3 interpreter when running this script.